src/radical/pilot/agent/scheduler/hombre.py

```diff
@@ -23,6 +23,15 @@
 class Hombre(AgentSchedulingComponent):
     '''
     HOMBRE: HOMogeneous Bag-of-task REsource allocator.  Don't kill me...
+
+    The scheduler implementation manages a set of resources (cores).  When it
+    gets the first scheduling request, it will assume that to bre representative
```

Comment:
Just a typo here (bre --> be, I assume).

Reply:
(This is the *only* comment I have. everything else is good, so I will accept, assuming you will fix it~).